views/partials/_whats-new.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="app-whats-new">
+  <div class="app-width-container">
+    <div class="govuk-main-wrapper govuk-main-wrapper--l">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full-from-desktop">
```

Comment:
Another design decision but if we wanted to limit the width of the border to two thirds like we do with the containers below, we could use `.govuk-grid-column-two-thirds-from-desktop` like we do for the Community section. It would make the border on this container the same width as the ones below instead of going full width.

![Screenshot 2021-06-24 at 16 21 41](https://user-images.githubusercontent.com/5007934/123290389-23fae800-d509-11eb-88f3-a5b881f6e4ed.png)

Reply:
Actually, I've just realised that the background colour on this container needs to be full width! So we could set the border on a `..govuk-grid-column-two-thirds-from-desktop` inside of a full width container.